addOns/network/gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=0.7.0
+version=0.6.0
```

Comment:
This will need to be reverted

Reply:
I can fix that one, I probably shoulda noticed when I rebased and squashed.